src/modules/85-cv/components/ExecutionVerification/components/LogAnalysisContainer/components/ClusterTypeFiltersForLogs.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback } from 'react'
+import { Checkbox, Container, Layout, MultiSelectDropDown, MultiSelectOption } from '@harness/uicore'
+import { useStrings } from 'framework/strings'
+import type { ClusterTypeFiltersForLogsProps } from './ClusterTypeFiltersForLogs.types'
+import { getDropdownItems, getFilterDisplayText } from '../../DeploymentMetrics/DeploymentMetrics.utils'
+import { EventTypeFullName, getClusterTypes } from '../LogAnalysis.utils'
+import css from './ClusterTypeFiltersForLogs.module.scss'
+import filterStyle from '../../DeploymentMetrics/DeploymentMetrics.module.scss'
+
+const ClusterTypeFiltersForLogs: React.FC<ClusterTypeFiltersForLogsProps> = ({
+  clusterTypeFilters,
+  onFilterChange,
+  selectedNodeName,
+  nodeNames,
+  nodeNamesLoading,
+  nodeNamesError,
+  handleNodeNameChange
+}) => {
+  const { getString } = useStrings()
+
+  const checkboxItems = getClusterTypes(getString)
+
+  const getFilteredText = useCallback((selectedOptions: MultiSelectOption[] = [], filterText = ' '): string => {
+    const baseText = getString(filterText)
+    return getFilterDisplayText(selectedOptions, baseText, getString('all'))
+  }, [])
```

Comment:
Codacy has a fix for the issue: [React Hook useCallback has a missing dependency: 'getString'. Either include it or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232318)
```suggestion
  }, [getString])
```